### PR TITLE
Scripts for twist evaluation and eagleye evaluation

### PR DESCRIPTION
**Pull request by ppaul12**
- changed `main.py` to `pose_main.py`
- changed `config/evaluation.yml` to `config/evaluation_pose.yml`
- added scripts for eagleye evaluation to sub_scripts
- added scripts for twist evaluation as `twist_main.py`

### CODE REVIEW

File: config/evalutaion_eagleye.yaml
Comment (rsasaki0109) [inline config/evalutaion_eagleye.yaml:1]: There was a typo. It should be `evaluation_eagleye.yaml`, not `evalutaion_eagleye.yaml`.
There are other places where it says evalutaion.

**Review by rsasaki0109 — APPROVED** [review]

**Comment by rsasaki0109** [comment]
I have tested the functionality of `scripts/twist_main.py` and `sub_scripts/sub_eagleye_evaluation.py`, and they seem to be working without any issues.

File: config/evalutaion_eagleye.yaml
Comment (ppaul12) [reply config/evalutaion_eagleye.yaml:1]: Thank you for your review. I have fixed all two typos in those file names.
- `evalutaion_eagleye.yaml` -> `evaluation_eagleye.yaml`
- `evalutaion_twist.yaml` -> `evaluation_twist.yaml`

**Comment by KOKIAOKI** [comment]
## Add Releases
Release number will be 1.0.0 in this commit
Add Release tag 
Follow the Release number rule since this commit.

## Remove following files and folders because they don't seem to be used
config/adjust_time_stamp.yaml
sample_data/adjust_time_test_data
sample_data/config/adjust_time_stamp.yaml

sub_scripts/develop/
sub_scripts/adjust_time_stamp.py

Remove adjust_time_stamp item in README

## Simplify README
title
summary 
Main: Compare multiple localization results. CSV file and ros2 bag file are available. Eagleye: Eagleye evaluation. 
Sub: Evaluation with localization performance in Autoware.

1.Installation
2.Main Evaluation (pose_main.py)
- pose_main.py
- twist_main.py  

3.Sub Evaluation (second README file) 
A. NDT performance in Autoware NDT (Evaluate TP, NVTL, execution time, iteration and error ellipse with ros2 bag files)
B. sub eagleye evaluation
C. Create CSV file with covariance column from ros1 bag file
4.Test run (click open)
test command (Please fix)
graphs

Detail explanation should be hided with click button

## Add explanation
twist.yaml

## Others
Save graphs before displaying graphs
Revise “TF” to “Adjustment”
If you have a time,
- simplify yaml setting by using array
- 
- add another topic in ros2bag input. e.g. geometry_msgs/PoseStamped (Future task)

**Comment by KOKIAOKI** [comment]
My suggestion of new yaml format 
[evaluation_pose.txt](https://github.com/MapIV/localization_evaluation_toolkit/files/11228410/evaluation_pose.txt)

**Review by KOKIAOKI — APPROVED** [review]
Thank you for the additional fix.

**Pull request merged**